### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by DmitriyMusatkin**
*Issue #, if available:*

*Description of changes:*
Add basic README

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+### Building
 
-## License
+CMake 3.0+ is required to build.
+
+`<install-path>` must be an absolute path in the following instructions.
+
+
+#### Building aws-c-s3 and Remaining Dependencies
```
Comment (TingDaoK) [inline README.md:19]: ~aws-c-s3~ -> aws-c-sdkutils

**Review by TingDaoK — APPROVED** [review]
fix and ship

File: README.md (hunk 1 above)
Comment (TingDaoK) [reply README.md:19]: actually the `Remaining Dependencies` are not useful here as well

Comment (DmitriyMusatkin) [reply README.md]: ah, yeah copy-paste error. fixed in next commit

**Review by graebm — APPROVED** [review]

**Pull request merged**